### PR TITLE
docs filter price

### PR DESCRIPTION
**Pull request by at-sonle**
## Description
docs filter by price
## Reference Requirements
- Task ID: TI-1

## Reference Isuees
#{issue_id}

## Migrations
 NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

### CODE REVIEW

**Comment by at-thaitran** [comment]
LGTM

File: docs/api-references/product.md [hunk 1]
```diff
@@ -239,6 +241,74 @@ Get product
...
+                "name": "Lavada Buckridge",
+                "price": "173.00",
+                "quantity": 0,
+                "category_id": 9,
+                "preview": "Non voluptate repellat sunt ipsa. Porro et distinctio est tenetur debitis reprehenderit amet qui. Odit aspernatur fuga nam quod. Assumenda alias pariatur repellat blanditiis repellat nobis.",
+                "description": "Dicta quae et dolorem temporibus dolorem voluptas. Et voluptatem consequuntur doloribus earum tempora quis sequi. Commodi consequuntur itaque rerum debitis consequatur.",
+                "avg_rate": 2.4,
+                "sum_rate": 3,
+                "total_rate": 13,
+                "created_at": "1982-07-13 03:40:36",
```
Comment (at-thinhnguyen) [inline docs/api-references/product.md]: xoá `deleted_at` đê
các trường `created_at` `updated_at` ko dùng thì xoá nhá

File: docs/api-references/product.md [hunk 2]
```diff
@@ -239,6 +241,74 @@ Get product
...
+                    "parent_id": 7,
+                    "created_at": "2018-06-14 03:52:27",
+                    "updated_at": "2018-06-14 03:52:27",
+                    "deleted_at": null,
+                    "level": 2
+                },
+                "images": [
+                    {
+                        "id": 1,
+                        "image": "default-product.jpg",
```
Comment (at-thinhnguyen) [inline docs/api-references/product.md]: trường này nếu bỏ dc thì bỏ

Comment (at-sonle) [reply docs/api-references/product.md]: ok anh

**Pull request merged**